oppijanumerorekisteri-service/src/main/java/fi/vm/sade/oppijanumerorekisteri/services/impl/OppijaTuontiAsyncServiceImpl.java

```diff
@@ -0,0 +1,31 @@
+package fi.vm.sade.oppijanumerorekisteri.services.impl;
+
+import fi.vm.sade.oppijanumerorekisteri.configurations.AsyncConfiguration;
+import lombok.RequiredArgsConstructor;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import fi.vm.sade.oppijanumerorekisteri.services.OppijaTuontiService;
+import fi.vm.sade.oppijanumerorekisteri.services.OppijaTuontiAsyncService;
+import org.springframework.transaction.annotation.Propagation;
+
+@Service
+@Transactional
+@RequiredArgsConstructor
+public class OppijaTuontiAsyncServiceImpl implements OppijaTuontiAsyncService {
+
+    private static final int ERÄKOKO = 1000;
```

Comment:
Vaikka java salliikin ääkköset muuttujien nimissä itse ainakin vältän näitä varmuuden vuoksi.